pkg/cmd/testdata/getenvoy.md

```diff
@@ -1,27 +1,51 @@
-+++
-title = "getenvoy"
-type = "reference"
-parent = "root"
-command = "getenvoy"
-+++
-## getenvoy
+% getenvoy 8
```

Comment:
need to polish the format template, but in general it is much better IMHO to do single-page markdown for such a small utility.

Reply:
I guess this breaks hugo builds

Reply:
we need +++ block

Reply:
magic info :) can't wait until we have things in the same repo

Reply:
agree even latest hugo can't handle this. will make a change

Reply:
this isn't really hugo related as normal markdown breaks rendering. It isn't broken, per-se, just it adds a useless line. I raised https://github.com/urfave/cli/issues/1275